BD-2094: Update Canvas FAQ with IAM info (#5366)
braze-inc/braze-docs: _docs/_user_guide/engagement_tools/canvas/faqs.md

- No further messages will be sent out, despite where a user is in the flow.
- **Exception:** Canvases with emails will not immediately stop. Once the send requests go to SendGrid, there is nothing we can do to stop them from being delivered to the user.

> If a user has entered a Canvas step with an in-app message before the Canvas was stopped, they will still receive the in-app message since in-app messages send upon the next session start.

Author: ```suggestion
#### In-app messages in Canvas 
In-app messages send upon the next session start. This means if the user enters the Canvas step before the Canvas is stopped, they will still receive the in-app message upon their next session start, as long as the in-app message hasn't expired yet.
```

> For Canvases that include in-app messages, because in-app messages send upon the next session start, if the user has entered the Canvas step before the Canvas was stopped, they will still receive the in-app message upon the next session start if the in-app message hasn't expired yet.

Author: ```suggestion
If a user has entered a Canvas step with an in-app message before the Canvas was stopped, they will still receive the in-app message since in-app messages send upon the next session start.
```

> For Canvases that include in-app messages, because in-app messages send upon the next session start, if the user has entered the Canvas step before the Canvas was stopped, they will still receive the in-app message upon the next session start if the in-app message hasn't expired yet.

Reviewer: Some suggestions for readability 🤔 
```suggestion
**In-app messages in Canvas:** In-app messages send upon the next session start. This means if the user enters the Canvas step before the Canvas is stopped, they will still receive the in-app message upon their next session start, as long as the in-app message hasn't expired yet.
```

Reviewer: Or maybe an h4 instead? "In-app messages in Canvases" or "Canvases with in-app messages" 🤔
